package.json

```diff
@@ -0,0 +1,50 @@
+{
+   "name": "glimmer.js",
+   "version": "0.8.0",
```

Comment:
Totally agreed that we should synchronize, but what I was actually getting at was that the "bumping" of the version number should be done all at the same time with the publish script (the last time I ran the `bin/publish` in glimmer-vm where one package had a different version number I got an error)